### DISCUSSION TITLE
Function "predict" in sahi/predict.py

### DISCUSSION THREAD

**Discussion by Smonkey123** [obss/sahi#1100] [Q&A] [answered]
There is a problem setting the dataset_json_path parameter type. Depending on the description and usage, dataset_json_path should be a string that specifies the path to the JSON file of the COCO format dataset. However, in the function definition, its type is incorrectly set to bool and the initial value is None

**Comment by fcakyon** [comment, marked as answer]
Thanks for mentioning @Smonkey123, it's fixed in the main branch! https://github.com/obss/sahi/pull/1111
